islandora_ocr.module

```diff
@@ -7,6 +7,10 @@
 
 // Constant (deprecated).
 define('ISLANDORA_OCR_TESSERACT_REQUIRED_VERSION', '3.02.02');
+// Constants (not deprecated).
+const ISLANDORA_OCR_GENERATE_OCR_RELS_STATEMENT = 'generate_ocr';
+const ISLANDORA_OCR_GENERATE_HOCR_RELS_STATEMENT = 'generate_hocr';
+const ISLANDORA_OCR_UPDATE_7001_TIMESTAMP = 'islandora_ocr_update_7001_generate_ocr_timestamp';
```

Comment:
Will wait for this legacy piece (this is 1/2 "ISLANDORA_OCR_UPDATE_7001_TIMESTAMP" references) to be removed before re-testing.